docs/dictionary.rst

```diff
@@ -33,17 +37,38 @@ Creative Pages
 In this context, Crative Pages correspond to sections of the site where content is provided by our authors, after review. Creative page has Categories (= Sekce) that serve to further classify the Creations. 
 
 * Rubriky (= Creative Pages)
+
 * Bestiář (= Monsters)
 * Dovednosti (= Skills)
 * Hřbitov (= Graveyard)
+* Články a Eseje (= Articles and Essays)
+* Galerie (= Gallery)
+* Fotogalerie (= Photogallery)
 * Alchymistické Předměty (= Alchemist Tools)
-* Hraničářská Kouzla (= Ranger Spell)
+* Alchymista (= Alchemist)
+* Hraničář (= Ranger)
+* Hraničářská Kouzla (= Ranger Spells)
+* Válečník (= Warrior)
+* Zloděj (= Rogue)
+* Kouzelník (= Wizard)
+* Kouzelnická kouzla (= Wizard Spells)
+* Nové rasy (= New Races)
+* Nová povolání (= New Classes)
+* Rozvoj DrD (= DrD Expansion)
```

Comment:
Since DrD is czech abbreviation, wouldn't it be better to have something like "System expansion" instead?

Reply:
Problem is this section is usually not about the system and more about the game theory and world design. Just looking at subcategories:
Alternate rules
Armies and military
Economy and trade
Supernatural
Politics and diplomacy
Worlds and settings
RPG theory
Other
Honestly even the czech name "Rozvoj DrD" is not great, but naming it in english is even trickier.
(BTW just looking at some creations I could probably tell that this category was created in a reaction to several articles about world building and game theories in ČaE in order to differentiate between artistic articles and game related articles / expansions.)
What does @Almad think?

Reply:
So potentially some new and better name even in Czech?

Reply:
@Almad yea, maybe "Rozvoj hry", because you can use that for all RPG systems in general. And in english it could be "game expansion" (best term would be "game development" but that is already used in a different context), but we already have "expansions" for Doplňky. (Unless we return to Supplements.)